### PR TITLE
Fix a broken link in the manual

### PR DESCRIPTION
**Pull request by novakboskov**

### CODE REVIEW

**Comment by Malabarba** [comment]
Thanks. Could you just change the commit message to "Fix a broken link in the readme"?

File: doc/installation.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Stable by adding this to your Emacs initialization:
 ## CIDER's nREPL middleware
 
 Much of CIDER's functionality depends on the presence of CIDER's own
-[nREPL middleware][cider-nrepl]. Starting with version 0.11, When `cider-jack-in` (<kbd>C-c M-j</kbd>) is
+[nREPL middleware](https://github.com/clojure-emacs/cider-nrepl). Starting with version 0.11, When `cider-jack-in` (<kbd>C-c M-j</kbd>) is
```
Comment (bbatsov) [inline doc/installation.md]: Add the link to the bottom of the file as a named link.

Comment (novakboskov) [inline doc/installation.md]: To refer to "Using embedded nREPL server" section?

Comment (bbatsov) [inline doc/installation.md]: Just change the commit message for now.

**Comment by bbatsov** [comment]
readme -> manual

**Pull request merged**